stdlib/public/Concurrency/Task.swift

```diff
@@ -323,6 +323,21 @@ func isTaskCancelled(_ task: Builtin.NativeObject) -> Bool
 @_silgen_name("swift_task_runAndBlockThread")
 public func runAsyncAndBlock(_ asyncFun: @escaping () async -> ())
 
+@_silgen_name("swift_task_asyncMainDrainQueue")
+public func _asyncMainDrainQueue() -> Never
+
+public func _runAsyncMain(_ asyncFun: @escaping () async throws -> ()) {
+  let _ = Task.runDetached {
```

Comment:
No need to modify this PR, but we should track this TODO in radar: Once we have custom executors and the ability to annotate detached tasks with a custom executor, we probably need this task to be marked as running on the main executor.